src/Neo.VM/Types/StackItem.Vertex.cs

```diff
@@ -120,7 +126,14 @@ internal class ObjectReferenceEntry
         /// Use this method when you need a hash code for a StackItem.
         /// </summary>
         /// <returns>The hash code for the StackItem.</returns>
-        public override int GetHashCode() =>
-            HashCode.Combine(GetSpan().ToArray());
+        public override int GetHashCode()
+        {
+            if (_hashCode == 0)
+            {
+                using Murmur32 murmur = new(s_seed);
+                _hashCode = BinaryPrimitives.ReadInt32LittleEndian(murmur.ComputeHash(GetSpan().ToArray()));
+            }
+            return _hashCode;
```

Comment:
The buffer can change, I still think that we can just return the Length, what do you think @Jim8y ?

Reply:
well your right @shargon about that, I just copied `ByteString` didn't think of that before. `ByteString` is the same way as well. I can change them both it you want @shargon.

Reply:
It may better to calculate the actual hash value. If content is long, it can only calculate a prefix of the content.